apps/api/src/app/events/usecases/send-message/digest/digest.usecase.ts

```diff
@@ -29,7 +32,7 @@ export class Digest extends SendMessageType {
     await this.createExecutionDetails.execute(
       CreateExecutionDetailsCommand.create({
         ...CreateExecutionDetailsCommand.getDetailsFromJob(command.job),
-        detail: 'Steps to get digest events found ' + nextJobs.length,
+        detail: DetailEnum.DIGESTED_EVENTS_PROVIDED + ' ' + nextJobs.length,
```

Comment:
@davidsoderberg I wonder about this section, in case we will need the data in a structured way in the UI. Won't it be better to have the specific enum as only the enum. And then save the raw information to be able to extract? In this scenario we won't be able to catch this specific execution detail when needed because it's ever changing :)

Reply:
I agree the only thing I see now is that the raw object might need a structure for it to be predictable for frontend to display info in it...